libsplinter/src/peer/connector.rs

```diff
@@ -24,29 +24,29 @@ use super::notification::PeerNotificationIter;
 use super::{EndpointPeerRef, PeerRef};
 use super::{PeerManagerMessage, PeerManagerRequest};
 
-/// The PeerLookup trait provides an interface for looking up details about individual peer
+/// The `PeerLookup` trait provides an interface for looking up details about individual peer
 /// connections.
 pub trait PeerLookup: Send {
     /// Retrieves the connection id for a given peer id, if found.
     ///
     /// # Errors
     ///
-    /// Returns a PeerLookupError if the connection id cannot be retrieved.
+    /// Returns a `PeerLookupError` if the connection id cannot be retrieved.
```

Comment:
Should be ID in order to follow Splinter Rustdoc style guidelines and be consistent with the Rustdoc contents elsewhere in this PR.  (I didn't mark the other occurrences of "id".)